.github/workflows/build-and-test.yml

```diff
@@ -30,4 +30,15 @@ jobs:
     - name: Self Report
       # Generates a report based on the logged data from verifying itself.
       # This is both a guard against unstable verification and a smoke test of the tool itself.
-      run: dotnet run --project src -- summarize-csv-results --max-resource-count 1000000 .
+      run: dotnet run --project src -- summarize-csv-results --max-resource-count 1500000 .
+    - name: Stability Analysis
+      # Analyzes the stability of verification by running it several
+      # more times and calculating statistics
+      #
+      # Runs Dafny directly because it doesn't seem like we can pass
+      # multiple arguments to it through MSBuild.
+      run: |
+        for iter in 1 2 3 4 5; do
+          (cd src && dafny /compile:0 /timeLimit:30 /verificationLogger:csv /randomSeed:$iter Main.dfy)
```

Comment:
Why not `dotnet run` with the Dafny directory?

Reply:
This is slightly simpler?

Reply:
OK.  Bu tare we sure it picks up the right Dafny?  I guess there's only one anyway.

Reply:
Yeah, there should only be one.